raisinbread/instruments/eeg_acquisition_form.rules

```diff
@@ -14,21 +14,29 @@ placement_comment{-}Required when placement is not available{-}placement_rating{
 acquisition_software_other{-}Required when acquisition software is other{-}acquisition_software{@}=={@}other
 recording_reference_other{-}Required when recording reference is other{-}recording_reference{@}=={@}other
 eog_channels_other{-}Required when EOG channels is other{-}eog_channels{@}=={@}other
-artefacted_channels{-}Not Required{-}artefacted_channels{@}=={@}NEVER_REQUIRED
+artefacted_channels{-}Required when artefacted channels are present{-}artefacted_channels_present{@}=={@}yes
 hardware_filters_applied{-}Not Required{-}hardware_filters_applied{@}=={@}NEVER_REQUIRED
 software_filters_applied{-}Not Required{-}software_filters_applied{@}=={@}NEVER_REQUIRED
 resting_state_eeg_incomplete_reason{-}Required when resting state acquiring is incomplete{-}resting_state_eeg_acquired{@}=={@}incomplete
 resting_state_subject_mood_rating_comments{-}Not Required{-}resting_state_subject_mood_rating_comments{@}=={@}NEVER_REQUIRED
 resting_state_comments{-}Not Required{-}resting_state_comments{@}=={@}NEVER_REQUIRED
 task1_eeg_incomplete_reason{-}Required when task 1 acquiring is incomplete{-}task1_eeg_acquired{@}=={@}incomplete
-task1_subject_mood_rating_comments{-}Not Required{-}task1_subject_mood_rating_comments{@}=={@}NEVER_REQUIRED
+task1_attempts{-}Required when task 1 is acquired{-}task1_eeg_acquired{@}=={@}complete|incomplete
+task1_subject_mood_rating{-}Required when task 1 is acquired{-}task1_eeg_acquired{@}=={@}complete|incomplete
```

Comment:
```suggestion
task1_subject_mood_rating{-}Required when this task is acquired{-}task1_eeg_acquired{@}=={@}complete
```
for all tasks:
- [ ]  I think the mood should be required only when complete.  
- [ ] change `task N` to `this task`